hbase-client/src/main/java/org/apache/hadoop/hbase/client/Admin.java

```diff
@@ -513,6 +513,15 @@ default void modifyColumnFamily(TableName tableName, ColumnFamilyDescriptor colu
    */
   void flush(TableName tableName) throws IOException;
 
+  /**
+   * Flush a table with specified column family. Synchronous operation.
```

Comment:
Let's not leave any room for confusion: `Flush the specified column family stores on all regions of the passed table. This runs as a synchronous operation.`

Reply:
Will fix later.
Thanks.